io/src/main/scala/sbt/internal/io/Retry.scala

```diff
@@ -24,7 +24,13 @@ private[sbt] object Retry {
   private[sbt] def apply[@specialized T](
       f: => T,
       limit: Int,
-      excludedExceptions: Class[_ <: IOException]*
+      excludedExceptions: Class[_ <: IOException]*,
+  ): T = apply(f, limit, 100, excludedExceptions: _*)
+  private[sbt] def apply[@specialized T](
+      f: => T,
+      limit: Int,
+      sleepDuration: Long,
```

Comment:
I feel like this should use Duration class, and have a val for 100ms so we don't have to allocate it each time.

Reply:
That seems like overkill for a private api with defaults.

Reply:
or at least change the param name to `sleepInMillis`

Reply:
done